TLM/TLM/UI/WhatsNew/WhatsNewPanel.cs

```diff
@@ -1,6 +1,7 @@
 namespace TrafficManager.UI.WhatsNew {
     using System;
     using System.Collections.Generic;
+    using System.Net.Configuration;
```

Comment:
?

Reply:
hehe, probably I made a typo and it automatically imported this thing 😂